### PR TITLE
Specify model name on initialize and fail import notification messages

### LINKED ISSUE #6180: Skins and Beatmaps "Import" notifications are identical
**Describe the issue:** Starting an import from scratch, Lazer starts an import for beatmaps and another import for skins, but both of them start with "_Import is initialising..._", I almost thought it was a bug (like if it started two imports for beatmaps):

image

**Proposal designs of the feature:** I guess it's better for skins a message like "_Skins import is initialising..._", and the same thing for beatmaps: "_Beatmaps import is initialising..._", and/or with a different icon (still missing) based on the import.

### PR DESCRIPTION
**Pull request by frenzibyte**
Closes #6180

### CODE REVIEW

**Review by bdach — COMMENTED** [review]
If you're doing `Humanize()` everywhere, you missed one on [line 134](https://github.com/ppy/osu/blob/8300e86f20815f9add57bdc4132437186af7346d/osu.Game/Database/ArchiveModelManager.cs#L134) and another on [line 157](https://github.com/ppy/osu/blob/8300e86f20815f9add57bdc4132437186af7346d/osu.Game/Database/ArchiveModelManager.cs#L157). Other methods also use `HumanisedModelName` raw, might be actually preferable to use Humanizer [in the getter itself](https://github.com/ppy/osu/blob/8300e86f20815f9add57bdc4132437186af7346d/osu.Game/Database/ArchiveModelManager.cs#L599).

Not an expert on Humanizer, but we might want to do `Pluralize()` instead wherever plurals are involved?

**Comment by smoogipoo** [comment]
+1 to `Pluralize`

**Comment by iiSaLMaN** [comment]
> Not an expert on Humanizer, but we might want to do `Pluralize()` instead wherever plurals are involved?

I actually used humanize to make the text start with a capital letter, will humanize the entire text instead.

**Comment by peppy** [comment]
I find the `Humanize` call on the whole string a bit confusing here. Would probably rather it be local to the model name call.

**Comment by iiSaLMaN** [comment]
> I find the `Humanize` call on the whole string a bit confusing here. Would probably rather it be local to the model name call.

Resolved.

File: osu.Game/Database/ArchiveModelManager.cs [hunk 1]
```diff
@@ -110,7 +111,7 @@ public Task Import(params string[] paths)
         protected async Task Import(ProgressNotification notification, params string[] paths)
         {
             notification.Progress = 0;
-            notification.Text = "Import is initialising...";
+            notification.Text = $"{HumanisedModelName.Humanize().Pluralize(paths.Length == 1)} import is initialising...";
```
Comment (peppy) [inline osu.Game/Database/ArchiveModelManager.cs]: This should not be plural. Also you probably want `.Humanize(LetterCasing.Title)`.

File: osu.Game/Database/ArchiveModelManager.cs [hunk 2]
```diff
@@ -146,7 +147,7 @@ protected async Task Import(ProgressNotification notification, params string[] p
 
             if (imported.Count == 0)
             {
-                notification.Text = "Import failed!";
+                notification.Text = $"{HumanisedModelName.Humanize().Pluralize(paths.Length == 1)} import failed!";
```
Comment (peppy) [inline osu.Game/Database/ArchiveModelManager.cs]: ditto, pluralize will be incorrect in every case.

**Review by peppy — APPROVED** [review]

**Pull request merged**